docs/cryptodoc/src/05_09_kyber.rst

```diff
@@ -3,14 +3,6 @@
 Kyber
 =====
 
-.. todo::
-
-   This documentation is outdated (and potentially too detailed).
-   It should be updated as soon as those pull requests are merged:
-
-     * https://github.com/randombit/botan/pull/4024
-
-   Until then, I've removed some of the source links to pass CI.
```

Comment:
This was postponed to 3.6.0 but https://github.com/randombit/botan/pull/4024 was merged in the meantime. I.e. we'll have to tackle the overhaul of the Kyber/Dilthium chapters rather soon.